newrelic.js

```diff
@@ -0,0 +1,11 @@
+const cfenv = require("cfenv")
+const appEnv = cfenv.getAppEnv()
+const newRelicEnv = appEnv.getServiceCreds("analytics-reporter-new-relic")
```

Comment:
We have an [environment service in Federalist](https://github.com/18F/federalist/blob/staging/services/environment.js) that handles this. May work here.

Additionally, I'm looking now and thinking the New Relic configs may fit comfortably in the existing `config.js`. I think this is the direction I'm leaning.